<launch>
	<!-- <include file="$(find pharos_localization)/launch/utm_odometry.launch" /> -->
	<arg name="initpose_fix" default="true" />
	<arg name="bagfile" default="false" />

	<node name="hybrid_ekf" pkg="pharos_localization" type="hybrid_ekf.py" output="screen">
		<param name="vehicle_state_topic" value="/vehicle/state2016" />
		<param name="frame_id" value="/odom" />
		<param name="child_frame_id" value="/mcl_ekf" />
		<param name="mcl_odom_topic" value="/odom/mcl" />
		<param name="isbagfile" value="$(arg bagfile)" />


		<param name="relative_gps" value="true" />
		<param name="publish_odom_tf" value="true" />
		<param name="initpose_fix" value="$(arg initpose_fix)" />
	</node>
	<!-- <node name="node_lane_ekf" pkg="pharos_localization" type="lane_ekf.py" output="screen" />	 -->

</launch>